ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/HardFork/Combinator/Ledger.hs

```diff
@@ -732,6 +746,103 @@ shiftUpdate = go
           (eraIndexSucc ix)
           (eraIndexSucc ix')
 
+{-------------------------------------------------------------------------------
+  Extending
+-------------------------------------------------------------------------------}
+
+-- | Extend the telescope until the specified slot is within the era at the tip
+--
+-- TODO documentation (in particular that this now does ticking!)
+extendToSlot ::
+     forall xs. CanHardFork xs
+  => HardForkLedgerConfig xs
+  -> SlotNo
+  -> HardForkState LedgerState xs
+  -> LedgerResult (LedgerState (HardForkBlock xs)) (HardForkState LedgerState xs)
+extendToSlot  ledgerCfg@HardForkLedgerConfig{..} slot ledgerSt@(HardForkState st) =
```

Comment:
I anticipate we should be ticking N times if we do N translations. Do you agree?

And I think I see that natural pairing in `howExtend`. Would you add a comment there highlighting it?

Reply:
(Just a note to self: GH labels this Outdated, but the comment isn't there yet.